_includes/job-posting.md

```diff
@@ -0,0 +1,9 @@
+<!-- prettier-ignore-start -->
+
+{% if post.url %}
+[{{ post.title }}]({{ post.url }})
+{%- else -%}
+{{ post.title }}
+{%- endif -%}&nbsp;- {{ post.text }}
```

Comment:
I couldn't figure out another way to ensure there was a space between the post title and the post text other than using `&nbsp;`. A plain space was being eaten by the markdown parser.